lib/maze/server.rb

```diff
@@ -93,6 +93,14 @@ def start
               Logger: $logger,
               AccessLog: []
             )
+
+            # Mount a block to respond to all requests with status:200
+            server.mount_proc '/' do |_request, response|
+              $logger.info 'Received request on server root, responding with 200'
+              response.header['Access-Control-Allow-Origin'] = '*'
+              response.status = 200
```

Comment:
It's absolutely fine as it is, but would it be worth putting a "Hello from Maze Runner" or similar in the response body for a human reading it?